compiler/luci/service/src/Nodes/CircleFullyConnected.test.cpp

```diff
@@ -16,8 +16,10 @@
 
 #include "luci/Service/CircleNodeClone.h"
 
-#include <gtest/gtest.h>
+#include "luci/Service/CircleShapeInference.h"
 
+#include <gtest/gtest.h>
+#include <iostream>
```

Comment:
```suggestion

```

Reply:
Sorry, I checked the code but missed it.
Thanks for reviewing. :)